test/integration/testLargeResultSet.js

```diff
@@ -106,3 +107,43 @@ describe('Large result Set Tests', function ()
     });
   });
 });
+
+describe('SNOW-743920: Large result set with ~35 chunks', function () {
+  let connection;
+  const tableName = 'test_table';
+  const sourceRowCount = 251002;
+  const createTable = `create or replace table ${tableName} (data string)`;
+  const populateData = `insert into ${tableName} select randstr(20, random()) from table (generator(rowcount =>${sourceRowCount}))`;
+  const selectAllFromOrders = `select * from ${tableName}`;
```

Comment:
What do you mean by `orders`?

Reply:
it's already renamed to just `selectData`